### PR TITLE
[home-assistant] deprecate appdaemon and configurator sidecars

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

- Waiting on https://github.com/k8s-at-home/charts/pull/326
- VSCode seems like the popular choice over configurator

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/home-assistant/.helmignore [hunk 1]
```diff
@@ -19,5 +19,6 @@
 .project
 .idea/
 *.tmproj
+.vscode/
 # OWNERS file for Kubernetes
 OWNERS
```
Comment (bjw-s) [inline charts/home-assistant/.helmignore:24]: missing newline 😛

File: charts/home-assistant/CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +44,8 @@ Any pre-existing StatefulSet will have to be removed before upgrading due to a n
 
 This is the last version before starting this changelog. All sorts of cool stuff was changed, but only `git log` remembers what that was :slightly_frowning_face:
 
+[4.0.0]: https://github.com/k8s-at-home/charts/tree/home-assistant-3.1.0/charts/home-assistant
```
Comment (bjw-s) [inline charts/home-assistant/CHANGELOG.md:47]:
```suggestion
[4.0.0]: https://github.com/k8s-at-home/charts/tree/home-assistant-4.0.0/charts/home-assistant
```

File: charts/home-assistant/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0]
+
+### Removed
+
+- Appdaemon sidecar was removed and replaced by it's own chart at [charts/appdaemon](https://github.com/k8s-at-home/charts/tree/master/charts/appdaemon)
+- Configurator sidecar was removed in favor of the `code-server` sidecar. One configure sidecar to rule them all!
+
```
Comment (bjw-s) [inline charts/home-assistant/CHANGELOG.md:13]: Also the mariadb version was bumped, right?

File: charts/home-assistant/.helmignore (hunk 1 above)
Comment (onedr0p) [reply charts/home-assistant/.helmignore]: @carpenike crys

**Comment by onedr0p** [comment]
Updated!

**Review by bjw-s — APPROVED** [review]

**Pull request merged**